src/core/toga/widgets/multilinetextinput.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, id=None, style=None, factory=None,
         super().__init__(id=id, style=style, factory=factory)
 
         self._impl = self.factory.MultilineTextInput(interface=self)
-        self.value = initial
+        self.value = "initial"
```

Comment:
This doesn't seem right - the initial value is something the user will provide, not the actual literal "initial"...

Reply:
Thanks for catching my mistake! That must have been a late night :sleeping: